backend/src/main/java/com/rota/api/HelloController.java

```diff
@@ -0,0 +1,43 @@
+package com.rota.api;
+
+import com.rota.api.dto.HelloWorld;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+  private AtomicLong id = new AtomicLong();
+  private String template = "Hello %s!";
+
+  /**
+   * /hello will return "Hello World!".
+   * /hello?message=foo witll return "Hello foo!".
+   * @param message Message to print
+   * @return HelloWorld response DTO
+   */
```

Comment:
A little later and in a separate pull request we can add Swagger integration. It generates a webpage with all our endpoints and info of what's going in and out! https://swagger.io/tools/swagger-ui/ Is what we used on MTG for this. Later I'll try and find some better resources for that.